### PR TITLE
Adding minutes from 2017-08-31 meeting

### PR DESCRIPTION
**Pull request by renrutnnej**
Please review and comment with corrections, suggestions.

### CODE REVIEW

File: meetings/2017-08-31.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+### Internationalization/Localization Working Groups [#114](https://github.com/nodejs/community-committee/issues/114)
+* Question: It appears the work of those working groups is technical in terms of code/documentation so should they fall under the TSC as opposed to the CommComm?
+  * For the most part, the work is actually focused around translation from what’s there in English into the local languages. The translation is a community aspect and accessibility is community focus. There’s currently no effort to bring these translation efforts together globally. 
+* If we agree to this, we need to insure that someone within CommComm is keeping tabs on and getting these groups what they need, a liaison to the Internationalization/Localization working group. 
+* CommComm reaches consensus (no objections) to move forward with bringing the Internationalization/Localization working group under the CommComm. 
+* There is an Internationalization working group repo, but not an active group (I18n).
+
+Action items:
+* Rachel and Rich volunteer to meet with Tracy and members currently working on translation efforts to start the transition.
```
Comment (RichardLitt) [inline meetings/2017-08-31.md]: Richard, if possible.

**Comment by RichardLitt** [comment]
(Who is currently in charge of merging? Can we just merge with multiple approvals?)

**Comment by bnb** [comment]
@RichardLitt based on the [Collaborator's Guide](https://github.com/nodejs/community-committee/blob/master/COLLABORATOR_GUIDE.md#accepting-modifications) section on accepting modifications, we usually wait 48-72 hours for non-trivial changes. 

I that believe that because we have several approvals (once your requested change is made), and we can probably merge at that point given this is more of a baseline documentation rather than something along the lines of a policy change or other truly substantial change.

**Comment by bnb** [comment]
Merging, since this has multiple +1s and no -1s. Thank you @renrutnnej 🙏

**Pull request merged**